data/pt_PT/resources/lang/messages_pt_PT.properties

```diff
@@ -1,5 +1,5 @@
 Launcher.menuLanguagePacks=Pacotes de idiomas
 Launcher.appTitle=Instalador de suporte para o idioma português (variante Europeia)
-Launcher.appTitle_TOOLTIP=Instala suporte para o idioma português (variante Europeia) no Pentaho 5
+Launcher.appTitle_TOOLTIP=Instala suporte para o idioma português (variante Europeia)
 languagePack.title=Instalador de suporte para o idioma português (variante Europeia)
 languagePack.description=Instala suporte para o idioma português (variante Europeia) no Pentaho 5
```

Comment:
Can you please replace "Pentaho 5" with simply "Pentaho", i.e. omit the version number ?
The language pack will run on both on 5.x and 6.x